src/module/actor/actor.js

```diff
@@ -265,7 +265,7 @@ export class ICRPGActor extends Actor {
     else if (this.type === 'monster') mod += attribute + this.system[group].all + this.system.allRollsMod;
 
     // Only exception to mod: defense
-    if (name === 'defense' && mod >= 10) mod -= 10;
+    if (name === 'defense' && mod >= 10) mod -= (game.settings.get('icrpgme', 'defenseStart'));
```

Comment:
It might be worth updating the condition here as well since the original goal of my change was to prevent `mod` from going negative, which could happen if `defenseStart` was greater than 10.

Reply:
you are absolutely right 😄 

does this solve it?
https://github.com/ClipplerBlood/icrpgme/commit/b49663751ca400ec492ee5c4562ab2b79fad1e48

Reply:
@colinbate @mordachai 
I would like to thank you for your help 
I've added a small contributors section to the readme https://github.com/ClipplerBlood/icrpgme?tab=readme-ov-file#contributors
I hope it's ok with you! And sorry for not merging the PRs earlier, I was quite busy with other stuff and kinda left this project behind...

Reply:
I must admit, I haven't used the system or played ICRPG much since shortly after making my PR, so I had to quickly scan the rules to refresh my memory. Looking back on my game in Foundry from back then, I may have been playing "incorrectly" since I seem to have been rolling Defense for the monsters when I should have just been using the Target.

However, the root cause of this stems from the fact that monsters in the system have a defense stat that is just the modifier. Maybe monsters aren't meant to have a defense stat at all. Technically the PC defense stat is just the CON + loot bonus, the +10 is only there for the target of enemy attacks. I know that some people like to have negative modifiers, so my suggestion would be not to have the +10 as part of the defense stat (for PCs) at all, but only add it when displaying the value in the sheet (in the little shield). That way in the dice roller it would show up as the proper modifier value.

This way people could can manage the modifier value directly. This should meet the needs of @mordachai and possibly remove the need to have `defenseStart` at all.